web/src/main/java/org/springframework/security/web/DefaultRedirectStrategy.java

```diff
@@ -97,4 +109,12 @@ protected boolean isContextRelative() {
 		return this.contextRelative;
 	}
 
+	public void setStatusCode(HttpStatus statusCode) {
```

Comment:
Please add JavaDoc describing how to use the method and when it comes in handy. It might be helpful to indicate that this method should be used to disambiguate 302 behavior (303 to rewrite the method, 307 to preserve it).